app/src/main/res/layout/fragment_summary_product.xml

```diff
@@ -136,6 +136,28 @@
 
             </RelativeLayout>
 
+            <Button
+                android:id="@+id/btn_eating"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="8dp"
+                android:padding="8dp"
+                android:background="@drawable/button_grey_border"
+                android:drawableLeft="@drawable/ic_add_24dp"
+                android:text="I'm eating this" />
```

Comment:
Can you extract the string to make it translatable ?

Reply:
Don't know why I forget this everytime :-P